auth-web/vue.config.js

```diff
@@ -16,35 +23,5 @@ module.exports = {
       warnings: true,
       errors: true
     }
-  },
-  pwa: {
```

Comment:
please test in local

Reply:
It was tested on DEV? 
![image](https://user-images.githubusercontent.com/3484109/177208892-a982e996-59bd-4d73-8bef-dcebebd02d2d.png)

Reply:
I did make some NGINX changes since then, they seem to work on local